omod/src/main/resources/config.xml

```diff
@@ -38,21 +38,23 @@
 		RedFlag.hbm.xml
 		MyCareHubSettings.hbm.xml
 		HealthDiary.hbm.xml
+        ConsentedPatient.hbm.xml
 	</mappingFiles>
 
 	<!-- Any required properties goes here-->
 	<globalProperty>
 		<property>mycarehub.api.url</property>
+        <defaultValue>https://mycarehub-demo.savannahghi.org/</defaultValue>

```

Comment:
Is there a way we can load these configs as environment variables?

Reply:
Also, this value should be the prod environment URL

Reply:
this was done @abiudrn

Reply:
Then revert the credentials change in L52 so that we load them when building the `.omod` locally. 

We can setup CI/CD in the future and load the variables as ENV variables